jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -97,6 +97,7 @@ class GemUtils {
                 // should already have taken care of, see #79
                 environment 'JBUNDLE_SKIP', true
                 environment 'JARS_SKIP', true
+                environment 'GEM_HOME', destDir.absolutePath
```

Comment:
I meant to set `GEM_PATH` to `destDir.absolutePath` - sorry not being clear enough